.github/workflows/dotnet-build.yml

```diff
@@ -0,0 +1,38 @@
+name: .NET Build
+
+on:
+  push:
+    paths:
+      - 'src/**'
+  pull_request:
+    types: [opened, edited, synchronize, reopened]
+    paths:
+      - 'src/**'
+  workflow_dispatch:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 6.0.x
+    
+    - name: Restore dependencies
+      working-directory: ./src/Ably.HealthCheck
+      run: dotnet restore
+    
+    - name: Build
+      working-directory: ./src/Ably.HealthCheck
+      run: dotnet build -c Release --no-restore
+
+    - name: Test
```

Comment:
It's a very small repo so perhaps caching NuGet packages has no sense, right?

Reply:
Agreed :)